src/beautifiers/black.coffee

```diff
@@ -37,3 +47,15 @@ module.exports = class Black extends Beautifier
       onStdin: (stdin) ->
         stdin.end text
     })
+    .then((stdout) =>
+        console.log stdout
+        if options.sort_imports
```

Comment:
Has this been tested? I do **not** expect this to work currently.

---

Please complete the following:
- [x] Add `sort_imports` option to `Python` language: https://github.com/Glavin001/atom-beautify/blob/master/src/languages/python.coffee#L47-L50
- [ ] Add `sort_imports` option to `Black` beautifier.
See https://github.com/Glavin001/atom-beautify/blob/master/src/beautifiers/js-beautify.coffee#L18-L25 for example:

```diff
-     Python: false
+    Python: 
+      sort_imports: true
```

https://github.com/Glavin001/atom-beautify/blob/65cc353080d528e77ea4821cd2fd94d7aec0b72d/src/beautifiers/black.coffee#L39

Reply:
I'm not sure I follow the request. Black sorts the way it thinks is best--it doesn't have a sort option. This PR is simply to make black work with isort, in a similar manner to the autopep8 extension.